README.md

```diff
@@ -14,4 +16,48 @@ An inter-agency lens would be applied to determining the selection criteria for
 iii) Pre-arranged finance (the money).
 CERF set aside around $5 million [pending final plan and confirmation] for anticipatory action for floods in
 Bangladesh. This funding will become available immediately once the defined trigger is reached to active the actions described above
-In addition, the pilot seeks to amplify and coordinate similar anticipatory action pilots at the agency scale, including from WFP, IFRC, and others.
\ No newline at end of file
+In addition, the pilot seeks to amplify and coordinate similar anticipatory action pilots at the agency scale, including from WFP, IFRC, and others.
+
+## Analysis
+
+The analysis within this repository contains two components. 
+
+1. Processing FFWC and GLOFAS forecasting data, used to trigger this pilot's anticipatory action. 
+2. Calculating historical estimates of flooding extent over time in five high priority districts (Bogra, Gaibandha, Jamalpur, Kurigram, Sirajganj). This work is largely based on an analysis of Sentinel-1 SAR imagery in Google Earth Engine, accessible [here](https://code.earthengine.google.com/0fe2c1f3b2cf8ef6fe9aa81382b00191). 
+
+## Structure of this repository 
+
+The content within this repository is structured as follows: 
+
+```
+├── data
+│   ├── processed          <- Data that has been processed or transformed in some way.  
+│   └── raw                <- Original, immutable data. 
+│
+├── notebooks              <- Jupyter notebooks that contain a walkthrough of data analysis steps. 
+│
+├── results                <- Final results. 
+│   └── plots              <- Output figures. 
+│
+├── scripts                <- Scripts to perform data processing and analysis. 
+│   │
+│   ├── d01_data           <- Scripts to load in datasets. 
+│   ├── d02_processing     <- Scripts to perform basic cleaning and preprocessing on data.
+│   ├── d03_analysis       <- Scripts to conduct more in-depth analysis of data
+│   └── d04_visualization  <- Scripts to create visualizations. 
+│
+├── README.md              <- Description of this project.
+└── environment.yml        <- Contains dependencies to set up a conda environment. 
+
+```
+
+Note that larger raw and processed data files are currently not included within this repository. 
```

Comment:
Are they needed to run the scripts? If so, maybe give instructions which files should be downloaded by the user and in which folder